maintut.go

```diff
@@ -2,20 +2,23 @@ package main
 
 import (
 	"fmt"
-	"github.com/Harkishen-Singh/GO-file-database/gobase"
-	// "./gobase"
+	// "github.com/Harkishen-Singh/GO-file-database/gobase"
+	"./gobase"
```

Comment:
this will break the semaphore build.
replace it with ```"github.com/Harkishen-Singh/GO-file-database/gobase"```

Reply:
Yaa.. sorry for the mistake.